libs/net/models/Areas/Editor/Models/AVOverview/AvOverviewSectionModel.cs

```diff
@@ -118,6 +118,8 @@ public AVOverviewSectionModel(Entities.AVOverviewTemplateSection entity) : base(
             SortOrder = model.SortOrder,
             Version = model.Version ?? 0
         };
+        if (!string.IsNullOrEmpty(model.UpdatedBy)) entity.UpdatedBy = model.UpdatedBy;
```

Comment:
Why are you doing this?  We set these values automatically already.  Refer to the `TNOContext.SaveChanges()` code.

Reply:
It always shows missing this info for some reason when in local debugging mode, however, the change is unnecessary and now removed, thanks